.github/ISSUE_TEMPLATE/documentation.yml

```diff
@@ -0,0 +1,44 @@
+name: "\U0001F99F Documentation issue"
+description: Tell us about issues like as outdated or insufficient documentation here
+title: "[docs] Change title"
```

Comment:
in PR titles we have `[documentation]` so maybe use the same one here?

Reply:
Thanks, updating.